utils/releaser/src/ReleaseWorker/AbstractShopsysReleaseWorker.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\ReleaseWorkerInterface;
+use Symplify\MonorepoBuilder\Release\Contract\ReleaseWorker\StageAwareReleaseWorkerInterface;
```

Comment:
I got an exception that `StageAwareReleaseWorkerInterface` does not exist - when I examine my vendor folder, there is `StageAwareInterface` for that purpose :slightly_smiling_face: